service/src/main/java/bio/terra/cbas/util/methods/GithubUrlDetailsManager.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.cbas.util.methods;
+
+public class GithubUrlDetailsManager {
```

Comment:
Correct me if I'm wrong, but I don't think we need this GithubUrlDetailsManager class anymore, assuming we can use GithubUrlComponents directly